src/st/st-box-layout.c

```diff
@@ -380,7 +380,7 @@ st_box_layout_paint (ClutterActor *actor)
 {
   StBoxLayout *self = ST_BOX_LAYOUT (actor);
   StBoxLayoutPrivate *priv = self->priv;
-  StThemeNode *theme_node = st_widget_get_theme_node (ST_WIDGET (actor));
+  StThemeNode *theme_node;
```

Comment:
I would initialize this with NULL, so we're not introducing overhead from referencing random memory and not using it.

Reply:
OK